uiowa/uiowa
# Update imagemagick settings with `convert` limits.

Resolves https://github.com/uiowa/uiowa/issues/7809




# How to test

sync a website local. i think everyone has imagemagick and convert as part of ddev... if not, this is available on develop (https://default.dev.drupal.uiowa.edu/media/add/image)
go to admin/config/media/image-toolkit#edit-imagemagick-exec and turn on the debugging information (temp, don't commit). this is already set on default.dev
go to media/add/image and upload an big image. you should see debug output with the convert command and the limit configuration strung together correctly as noted by the support ticket.
should be no error with upload. after upload you should be able to generate image styles by clicking the preview link.

<img width="1126" alt="Screenshot 2024-06-21 at 2 39 59 PM" src="https://github.com/uiowa/uiowa/assets/4663676/3a5725ec-41f4-4a5b-9d0d-456bb6501fbd">
